docs/workers.md

```diff
@@ -307,7 +307,12 @@ expose the `media` resource. For example:
            - media
 ```
 
-Note this worker cannot be load-balanced: only one instance should be active.
+Note that if running multiple media repositories they must be on the same server
+and you must configure which single instance to run the background tasks, e.g.:
```

Comment:
I'm finding this sentence a bit awkward to read, maybe the following is clearer?

```suggestion
Note that if running multiple media repositories they must be on the same server
and you must configure a single instance to run the background tasks, e.g.:
```